src/test/java/org/opensearch/security/SecuritySettingsTests.java

```diff
@@ -0,0 +1,454 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security;
+
+import com.google.common.collect.Lists;
+import org.junit.Assert;
+import org.junit.Test;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.security.ssl.util.LegacyOpenDistroSSLSecuritySettings;
+import org.opensearch.security.ssl.util.SSLSecuritySettings;
+import org.opensearch.security.support.LegacyOpenDistroSecuritySettings;
+import org.opensearch.security.support.SecuritySettings;
+
+import java.util.Map;
+
+public class SecuritySettingsTests {
+    
+    @Test
+    public void testLegacyOpenDistroSettingsFallback() {
+        Assert.assertEquals(
+                SecuritySettings.SECURITY_ADVANCED_MODULES_ENABLED.get(Settings.EMPTY),
```

Comment:
What is the purpose of the test? While getting setting from `Settings.EMPTY` even two unrelated settings will produce the same result.

Reply:
This is to test if the default value of `SecuritySettings` has fallback to the default value of `LegacyOpenDistroSecuritySettings`.